playbooks/infra-prometheus/setup-bind-exporter.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Install Docker
+  hosts: prometheus
+  become: True
+  roles:
+    - infra-ansible/roles/config-docker
```

Comment:
Could you add a README to the `playbooks` folder with instructions on how to install the required roles?
I also noticed that the `requirements.yml` is using `master` for `infra-ansible`. We probably should pin this to the latest tag/release of `infra-ansible` instead.

Reply:
Added the README to the directory. Changed the requirements.yml, to version: v1.0.16.
Seems that latest is not a valid tag on git repo, therefore I used the latest release.